### ISSUE TITLE
Improve web interface design

### ISSUE THREAD

**Issue by kuronosec** [kuronosec/zk-voto-digital#4]
Create a better design for the web interface

**Comment by mimisavage** [comment]
Let me try this one!

**Comment by drakkomaximo** [comment]
<img src="https://github.com/user-attachments/assets/2cf4109d-0f1c-4df8-8815-6eadca2d7ed0" alt="Logo" style="vertical-align: middle; margin-left: 10px; width: 20px; height: auto;"><br>
# [FE]: Improve web interface design 

Hi @kuronosec and everyone!, I'm a Dojo Coding member⛩️ 

I’d be happy to participate in this issue. With over 5 years of experience in frontend development, I’ve worked on similar projects and can ensure a clean, user-friendly, and well-structured implementation.

## Steps to Resolve:

<details>
  <summary><strong>1) Understand the project approach</strong></summary>
  <br>
  <ul>
    <li>I see that the project starts with a premise without a design, so we would need to meet to learn more about the project.</li>
  </ul>
</details>

<details>
  <summary><strong>2) Design</strong></summary>
  <br>
  <ul>
    <li>Create a minimalist design that reflects the desired functionality.</li>
  </ul>
</details>

<details>
  <summary><strong>3) Code Implementation</strong></summary>
  <br>
  <ul>
    <li>Translate the design into React code using the standards available at the time of development.</li>
  </ul>
</details>

<details>
  <summary><strong>4) Initial MVP</strong></summary>
  <br>
  <ul>
    <li>Show the design implemented in code and make necessary adjustments.</li>
  </ul>
</details>

<details>
  <summary><strong>5) Testing</strong></summary>
  <br>
  <ul>
    <li>Test to ensure that the information and designs do not break the user experience.</li>
  </ul>
</details>


---

**Estimated Time:** 3-5 days

Let me know if I can start working on this!

**Comment by kuronosec** [comment]
Hi @drakkomaximo please go ahead and thanks a lot! Let me know if you need more details about this issue or any doubt you have.

**Comment by drakkomaximo** [comment]
Hi @kuronosec I was reviewing your project and reading some specifications from the Dojo webpage. This is version 1 of the new webpage: https://zk-voto-digital.netlify.app/. At the moment, this is just a basic UX with a simple connection to MetaMask.

I created a new version because the current project needed updated package versions, and I used modern technology for the implementation. I couldn't run the previous version, so I developed this MVP to showcase some results.

This version still requires more functionality, but it represents the first step in that direction.

Note: I uploaded this new version to a separate repository before merging it into the corresponding branch, as I first need your approval since it introduces significant changes. Once I get the green light and it is merged into the proper branch, this repository will be deleted from my account.
